src/BugsnagUnity/Configuration.cs

```diff
@@ -10,14 +10,16 @@ public abstract class AbstractConfiguration : IConfiguration
 
     public const string DefaultSessionEndpoint = "https://sessions.bugsnag.com";
 
+    protected bool _autoNotify = true;
```

Comment:
I'm not keen on how this change leaks out to the abstract class which causes potential confusion with existing private `_autoNotify` variables in the Cocoa implementation.

How about we follow the pattern with the `apiKey` and pass this in as a parameter to `SetupDefaults`?